### PR TITLE
test(driver-adapters-wasm): unskip tests

### PR DESCRIPTION
**Pull request by millsp**
Corrects the mistakes and applies necessary changes to get the previously skipped tests to work.

closes https://github.com/prisma/team-orm/issues/552

### CODE REVIEW

File: .github/workflows/test.yaml
Comment (janpio) [inline .github/workflows/test.yaml:1]: The changes in this file should not be necessary as you are only changing files in the projects itself, right?

Comment (millsp) [reply .github/workflows/test.yaml:1]: Yes! I added this when I was still using the integration version.

File: driver-adapters-wasm/planetscale-cf-basic/prisma/schema.prisma [hunk 1]
```diff
@@ -6,8 +6,8 @@ generator client {
 
 datasource db {
   provider = "mysql"
-  // fake url to ensure the driver adapter is used
-  url      = "mysql://foo"
+  // TODO this url should not contain the db name
+  url      = "mysql://foo:bar@baz/eco-driver-adapters-planetscale-cf-basic?sslaccept=strict"
```
Comment (Jolg42) [inline driver-adapters-wasm/planetscale-cf-basic/prisma/schema.prisma:10]: TODO?

File: driver-adapters-wasm/planetscale-cf-basic/src/index.js [hunk 2]
```diff
@@ -137,22 +137,24 @@ export default {
...
-            name: 'Test 4',
-          },
-          update: {},
-          select: {
-            email: true,
-            age: true,
-            name: true,
-          },
-        }),
+        // Skipping this because of too many sub-requests (limit is 50 per fetch call)
```
Comment (Jolg42) [inline driver-adapters-wasm/planetscale-cf-basic/src/index.js:140]:
```suggestion
        // Skipping this because of too many sub-requests (limit is 50 per fetch call)
```
That's interesting!

File: driver-adapters-wasm/planetscale-vercel-nextjs-edgemw/prisma/schema.prisma [hunk 3]
```diff
@@ -6,8 +6,8 @@ generator client {
 
 datasource db {
   provider = "mysql"
-  // TODO: needs to be changed to a fake url to ensure the driver adapter is used
-  url      = env("DRIVER_ADAPTERS_PLANETSCALE_VERCEL_NEXTJS_EDGEMW_DATABASE_URL")
+  // TODO this url should not contain the db name
+  url      = "mysql://foo:bar@baz/eco-driver-adapters-planetscale-vercel-nextjs-edgemw?sslaccept=strict"
```
Comment (Jolg42) [inline driver-adapters-wasm/planetscale-vercel-nextjs-edgemw/prisma/schema.prisma:10]: Not sure what these TODO are for

File: driver-adapters-wasm/turso-cf-basic/src/index.js [hunk 4]
```diff
@@ -117,39 +141,43 @@ export default {
...
-            name: 'Test 4',
-          },
-          update: {},
-          select: {
-            email: true,
-            age: true,
-            name: true,
-          },
-        }),
+        // TODO skipped because of bug: "User Not Found"
```
Comment (Jolg42) [inline driver-adapters-wasm/turso-cf-basic/src/index.js:144]:
```suggestion
        // TODO skipped because of bug: "User Not Found"
```
🤔 Any idea why?

File: packagers/yarn3-workspaces-pnp/yarn.lock
Comment (Jolg42) [inline packagers/yarn3-workspaces-pnp/yarn.lock:1]: This looks like an unrelated deleted file?

Comment (millsp) [reply packagers/yarn3-workspaces-pnp/yarn.lock:1]: Not sure, I'll restore it

File: driver-adapters-wasm/turso-cf-basic/src/index.js (hunk 4 above)
Comment (millsp) [reply driver-adapters-wasm/turso-cf-basic/src/index.js:144]: I don't, I created an issue https://github.com/prisma/team-orm/issues/663

File: driver-adapters-wasm/planetscale-vercel-nextjs-edgemw/prisma/schema.prisma (hunk 3 above)
Comment (millsp) [reply driver-adapters-wasm/planetscale-vercel-nextjs-edgemw/prisma/schema.prisma:10]: We have an issue here https://github.com/prisma/team-orm/issues/662

File: driver-adapters-wasm/planetscale-cf-basic/prisma/schema.prisma (hunk 1 above)
Comment (millsp) [reply driver-adapters-wasm/planetscale-cf-basic/prisma/schema.prisma:10]: We have an issue here https://github.com/prisma/team-orm/issues/662
